### PR TITLE
K6 Testing

### PR DESCRIPTION
**Pull request by fschade**
add k6 as load tester

### CODE REVIEW

File: test/k6/.babelrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["@babel/env", "@babel/typescript"],
+  "plugins": [
+    "lodash",
+    "@babel/proposal-class-properties",
+    "@babel/proposal-object-rest-spread"
```
Comment (refs) [inline test/k6/.babelrc:6]: do we need this plugin even if we're not explicitly using the spread operator?

File: test/k6/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import resolve from '@rollup/plugin-node-resolve'
+import babel from 'rollup-plugin-babel'
+import { terser } from 'rollup-plugin-terser'
+import multiInput from 'rollup-plugin-multi-input';
+import path from 'path';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx']
```
Comment (refs) [inline test/k6/rollup.config.js:9]: `jsx` and `tsx` leftover from some react? I think we won't have view files here, orrrr?

**Review by refs — APPROVED** [review]
👍 yesssss thanks for the TS, long time no work with it <3.

File: test/k6/.babelrc (hunk 1 above)
Comment (fschade) [reply test/k6/.babelrc:6]: not right now, but soon we will... for example

```
http.puthttps://foo.bar", {headers: {...defaultHeaders(), foo: 'bar'}})
```

File: test/k6/rollup.config.js (hunk 2 above)
Comment (fschade) [reply test/k6/rollup.config.js:9]: yea its a left over, it can be removed

**Review by IljaN — COMMENTED** [review]
Maybe also integrate in to Makefile? (if possible)
:+1:

File: test/k6/.babelrc (hunk 1 above)
Comment (refs) [reply test/k6/.babelrc:6]: exactly, spread is a handy one 👍

**Comment by fschade** [comment]
> Maybe also integrate in to Makefile? (if possible)
> 👍

yea, sure... but first i need to get this landed as a base. later it gets an real integration

**Comment by refs** [comment]
@fschade you might want to add a changelog

File: test/k6/ tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (phil-davis) [inline test/k6/ tsconfig.json:1]: Is this file has a space at the beginning of the file name?
Was that intended?

**Comment by phil-davis** [comment]
What do we do about Codacy results on this sort of stuff? Merge anyway? Add it to an exclusion list? Adjust the formatting etc to make Codacy happy?

**Pull request merged**